### DISCUSSION TITLE
TreeTable filter method value type

### DISCUSSION THREAD

**Discussion by dmitrymizernik** [primefaces/community#804] [PrimeNG] [answered]
in 
https://github.com/primefaces/primeng/commit/4bdc2d87f9138377da12ed9e526882701cd86d2b#diff-e88f450bb0d4a5be3a4ed0a2ad65fc9c5253846241566164c1bbaec322bdba9eR1815 (row 1815)
 value type of filter method was changed from any to string, however, some of filters can work with arrays(for example in filter), could you please return type of 'value' parameter to any or extend it to array of strings at least?

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

We've updated the mentioned line https://github.com/primefaces/primeng/blob/2585e7c8f6c7f5a37d97e7712f9fb293bd4f6065/src/app/components/treetable/treetable.ts#L1855  

Thanks for reporting the issue!
